src/js/components/DataTable/Body.js

```diff
@@ -179,22 +180,41 @@ const Body = forwardRef(
     const [active, setActive] = React.useState();
     const [lastActive, setLastActive] = React.useState();
 
+    const selectRow = () => {
+      const primaryValue = data[active]?.[primaryProperty];
+      if (selected && selected.includes(primaryValue)) {
+        onSelect(selected.filter((s) => s !== primaryValue));
+      } else onSelect([...selected, primaryValue]);
+    };
+
+    const clickedRow =
```

Comment:
The term "clickedRow" seems to imply that the row was clicked. Perhaps "clickableRow" or "clickEnabled"?

Reply:
I can see that I like `clickableRow`